src/main/java/org/jetbrains/bsp/bazel/server/JavaBspServer.java

```diff
@@ -59,7 +50,8 @@ public JavaBspServer(
     // TODO(andrefmrocha): Remove this when kotlin is natively supported
     ActionGraphParser actionGraphParser =
         actionGraphResolver.parseActionGraph(
-            MnemonicsUtils.getMnemonics(targetsUnion, Lists.newArrayList(javac, kotlinc)));
+            MnemonicsUtils.getMnemonics(
```

Comment:
is this array modified? if not then it would be better to use ImmutableList.of(...)